src/components/DisplayField/index.tsx

```diff
@@ -68,7 +68,9 @@ export const DisplayField: React.FC<DisplayFieldProps> = ({
   }
 
   if (Number.isFinite(value)) {
-    displayText = new Intl.NumberFormat().format(Number(value));
+    displayText = new Intl.NumberFormat('de-DE', {
```

Comment:
Can we access the current application language here instead?

Reply:
see https://github.com/terrestris/shogun-gis-client/pull/1213